matplotcheck/vector.py

```diff
@@ -473,14 +473,18 @@ def assert_polygons(
     ):
         """Asserts the polygon data in Axes ax is equal to polygons_expected to decimal place dec with error message m
 		If polygons_expected is am empty list or None, assertion is passed
-		
+
 		Parameters
 		----------
 		polygons_expected: list of polygons expected to be founds on Axes ax
```

Comment:
@nkorinek please make sure each input parameter uses numpy style docs. so this should have the expected input type and then on the second line, the explanation of what that object is / does. please follow earthpy so we may need to update many of the docstrings.